File: SoloLearn Codes/Password Validation/PasswordValidation.cs
using System;
using System.Text.RegularExpressions;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine(new Regex("((?=.*[!@#$%&*])(?=.*[0-9]).{7,})").IsMatch(Console.ReadLine()) ? "Strong" : "Weak");
        }
    }
}

File: SoloLearn Codes/Password Validation/password_validation.swift
import Foundation

let str = readLine()!
print(try NSRegularExpression(pattern: "((?=.*[!@#$%&*])(?=.*[0-9]).{7,})")
    .matches(in: str, range: NSRange(location: 0, length: (str as NSString).length))
    .isEmpty ? "Weak" : "Strong")